force_wfmanager/central_pane/plot.py

```diff
@@ -111,22 +112,28 @@ def __plot_default(self):
         plot.set(title="Plot", padding=75, line_width=1)
 
         # Add pan and zoom tools
-        plot.tools.append(PanTool(plot))
-        plot.overlays.append(ZoomTool(plot))
+        scatter_plot.tools.append(PanTool(plot))
+        scatter_plot.overlays.append(ZoomTool(plot))
+
+        # Set the scatterplot's default selection marker invisible as it
+        # lead to artifacts on axis when switching between plotted cols
+        scatter_plot.set(selection_color=(0, 0, 0, 0),
+                         selection_outline_color=(0, 0, 0, 0))
 
         # Add the selection tool
         scatter_plot.tools.append(ScatterInspector(
             scatter_plot,
             threshold=10,
-            selection_mode="single",
+            multiselect_modifier=KeySpec(None, "shift"),
+            selection_mode="multi",
         ))
         overlay = ScatterInspectorOverlay(
             scatter_plot,
-            hover_color="blue",
+            hover_color=(0, 0, 1, 1),
```

Comment:
Is this tuple R,G,B,alpha? If it is would it make sense to have the hover selection be transparent and the actual selection be solid? Not exactly a big deal either way..

Reply:
I think this way round works, you can see explicitly which point you're about to select when you're hovering (if there are lots of points in close proximity), then then once selected, you get a larger marker that still lets you see nearby points, and their colour (for e.g. the next PR).